src/CozyUtils.spec.js

```diff
@@ -84,9 +84,87 @@ describe('CozyUtils', () => {
       cozyUtils.client.collection = jest.fn(() => ({
         find: findSpy
       }))
-      const result = await cozyUtils.getUpdatedContacts(
-        '2018-04-03T15:16:02.276Z'
+      const contactAccount = {
+        id: 'fakeAccountId',
+        lastSync: '2018-04-03T15:16:02.276Z',
+        shouldSyncOrphan: false
+      }
+      const result = await cozyUtils.getUpdatedContacts(contactAccount)
+      expect(result).toEqual([
```

Comment:
Is it relevant to test this? Unless I'm mistaken we're testing the mock return value, so essentially testing a testing component?

Reply:
it tests that we retrieve all contacts even if they are paginated